backend/coreapp/platforms.py

```diff
@@ -48,10 +49,15 @@ def to_json(
             "arch": self.arch,
             "has_decompiler": self.has_decompiler,
         }
-        if include_presets:
+        if include_presets:       
+            # Here we are filtering the presets based on the owner
+            # if owner is None it means the preset is a system one => we keep it
+            # if owner is equal to profile it means it has been created by the user => we keep it
+            # otherwise we do not use it
             ret["presets"] = [
                 PresetSerializer(p).data
                 for p in Preset.objects.filter(platform=self.id).order_by("name")
+                if p.owner is None or p.owner == profile
```

Comment:
Everyone should be able to see all presets. If I own project Banana and make a preset for it, I want everyone else on the site to be able to use that preset who's working on that  game. 

If there become too many presets to easily handle, we can make UI changes to make it easier to find the ones you want. but for now I think we don't need to filter like this. but I get your concern

Reply:
Good to know, I will restore previous code then